docs/source/usersguide/output/statepoint.rst

```diff
@@ -144,14 +142,10 @@ if (run_mode == MODE_EIGENVALUE)
 
         Unique identifier of the mesh.
 
-    **/tallies/meshes/mesh i/type** (*int*)
+    **/tallies/meshes/mesh i/type** (*char[]*)
```

Comment:
Is it true that ``i`` here is a monotonically increasing integer starting from 1? Or is it the unique mesh identifier specified by the user (and as indicated by the text above? If if is the user-specified identifier, then the loop here is mis-leading. If if is the internally generated index for the mesh, then that should be indicated for clarity.